docs/api-guide/fields.md

```diff
@@ -299,7 +299,7 @@ Corresponds to `django.db.models.fields.DateTimeField`.
 
 #### `DateTimeField` format strings.
 
-Format strings may either be [Python strftime formats][strftime] which explicitly specify the format, or the special string `'iso-8601'`, which indicates that [ISO 8601][iso8601] style datetimes should be used. (eg `'2013-01-29T12:34:56.000000Z'`)
+Format strings may either be [Python strftime formats][strftime] which explicitly specify the format, or the special strings `'iso-8601'`, which indicates that [ISO 8601][iso8601] style datetimes should be used (eg `'2013-01-29T12:34:56.000000Z'`), and `'iso-8601-strict'`, which indicates that [ISO 8601][iso8601] style datetimes without fraction seconds should be used (eg `'2013-01-29T12:34:56Z'`)
```

Comment:
Using 'strict' in the name is terse, but seems incorrect. 
- Fractional seconds are part of the standard - there isn't anything non-strict about them. 
- There isn't a 'strict' variant of the standard. Users searching for more information aren't going to find anything.